.agents/skills/iterative-refinement/SKILL.md

```
@@ -0,0 +1,149 @@
+  VALIDATE: ./scripts/validate.sh
+  Result: All gates pass, 92% coverage
+
+Final: Refactoring complete after 3 iterations
+```
+
+## Anti-Patterns to Avoid
+
+- Writing tests to pass buggy code
+- Skipping failing tests instead of fixing
+- Large refactors without incremental commits
+- Ignoring performance regressions
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)